app/models/immunisation_import.rb

```diff
@@ -90,6 +92,7 @@ def process!(patient_session:)
 
       rows
         .map(&:to_patient)
+        .reject(&:persisted?)
```

Comment:
If we're rejecting the ones that are already persisted does that mean we're intentionally not using the data in the import to update existing records with more recent data?

Reply:
Yup, I think that's the right assumption for now...

Reply:
Cool, yeah, happy with that I just wanted to check it was intentional.